docs/src/pages/demos/tables/EnhancedTable.js

```diff
@@ -306,6 +307,11 @@ class EnhancedTable extends React.Component {
                   </TableRow>
                 );
               })}
+              {emptyRows > 0 && (
```

Comment:
What about if we display 25 items? I think checking the page > 1 would be better.

Reply:
`emptyRows` is the number of "empty" rows on that page. If we display 25 items with 25 items per page, `emptyRows` will be 0 on the first page. If we display 25 items with 10 rows per page, it will be 0 on the first two pages and 5 on the last page.

Reply:
This line only ensures that the fill row is hidden when it's not needed. Otherwise, it would display two lines and a margin between the last row and the pagination.

Reply:
Oh… My formula doesn't work if the number of total rows is smaller than the number of rows per page. I'll fix that.

Reply:
What if we want to display 25 items but we only have 10. Won't the extra space be unneeded?

Reply:
Maybe, but it's also unneeded if you only have one item and show 5 per page. The filler ensures that the table always has the same height.

Reply:
@leMaik Ok, your call. I was being confused by the height of the table with the 25 option.

Reply:
@oliviertassinari I was too after your comment, but then… it's working as expected, it just looks strange.